sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/predicates.scala

```diff
@@ -202,7 +209,11 @@ case class InSubquery(values: Seq[Expression], query: ListQuery)
  */
 // scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "expr1 _FUNC_(expr2, expr3, ...) - Returns true if `expr` equals to any valN.",
+  usage = """
+    expr1 _FUNC_(expr2, expr3, ...) - Returns true if `expr` equals to any valN. Otherwise, if
+      spark.sql.legacy.inOperator.falseForNullField is false and any of the elements or fields of
+      the elements is null it returns null, else it returns false.
```

Comment:
I vaguely remember that multi-line string doesn't work with `ExpressionDescription`. Can you verify it with `DESCRIBE FUNCTION`?

Reply:
I think the point here is that only a string literal works, so it doesn't work concat and/or interpolation. This just puts the string on different lines, ie. the output is:
```
scala> sql("DESCRIBE FUNCTION IN").show(false)
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|function_desc                                                                                                                                                                                                                                                             |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|Function: in                                                                                                                                                                                                                                                              |
|Class: org.apache.spark.sql.catalyst.expressions.In                                                                                                                                                                                                                       |
|Usage: 
    expr1 in(expr2, expr3, ...) - Returns true if `expr` equals to any valN. Otherwise, if
      spark.sql.legacy.inOperator.falseForNullField is false and any of the elements or fields of
      the elements is null it returns null, else it returns false.
  |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Reply:
I'm wondering if we should use
```
"""
  |xxx
""".stripMargin
```

Reply:
that doesn't work. We cannot do `stripMargin`. We can just put a string.